pkg/apis/build/v1alpha1/build_pod.go

```diff
@@ -110,6 +110,51 @@ func (b *Build) BuildPod(config BuildPodImages, secrets []corev1.Secret, bc Buil
 		SubPath:   b.Spec.Source.SubPath, // empty string is a nop
 	}
 
+	bindingVolumes := []corev1.Volume{}
```

Comment:
can this be moved into a private method `setupBindings()`?

Reply:
done